src/um/ntlsa.rs

```diff
@@ -155,7 +155,7 @@ pub const SE_ADT_PARAMETERS_SEND_TO_LSA: ULONG = 0x00000002;
 pub const SE_ADT_PARAMETER_EXTENSIBLE_AUDIT: ULONG = 0x00000004;
 pub const SE_ADT_PARAMETER_GENERIC_AUDIT: ULONG = 0x00000008;
 pub const SE_ADT_PARAMETER_WRITE_SYNCHRONOUS: ULONG = 0x00000010;
-#[cfg(target_arch = "x86")]
+#[cfg(target_pointer_width = "32")]
```

Comment:
not sure I need to change this, mark it for now

Reply:
update to pointer width of 32 and 64, because this macro is about size of different structs which is not architecture specific

---
In reply to: [282984781](https://github.com/retep998/winapi-rs/pull/762#discussion_r282984781) [](ancestors = 282984781)